app/src/main/java/com/greenfox/opal/gitinder/response/StatusResponse.java

```diff
@@ -0,0 +1,22 @@
+package com.greenfox.opal.gitinder.response;
+
+public class StatusResponse {
+    String status;
+    String token;
```

Comment:
StatusResponse is the class we mentioned that stores all possible variables. Other responses will inherit from that.